### PR TITLE
Sandbox: Properly respect --log-level parameter

### PR DESCRIPTION
**Pull request by gerolf-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/logging/GlobalLogLevel.scala [hunk 1]
```diff
@@ -8,11 +8,12 @@ import org.slf4j.{Logger, LoggerFactory}
...
   def set(level: Level): Unit = {
-    LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME) match {
-      case rootLogger: ch.qos.logback.classic.Logger =>
-        rootLogger.setLevel(level)
+    val rootLogger = LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)
+    LoggerFactory.getILoggerFactory match {
+      case loggerContext: ch.qos.logback.classic.LoggerContext =>
         rootLogger.info(s"Sandbox verbosity changed to $level")
-      case rootLogger =>
+        loggerContext.getLoggerList.forEach(l => l.setLevel(level))
```
Comment (SamirTalwar-DA) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/logging/GlobalLogLevel.scala:15]:
```suggestion
        loggerContext.getLoggerList.forEach(_.setLevel(level))
```

File: ledger/participant-state/kvutils/app/src/main/resources/com/daml/ledger/participant/state/kvutils/app/logback.base.xml [hunk 2]
```diff
@@ -15,6 +15,22 @@
...
+    <logger name="com.digitalasset.platform" level="WARN" />
+
+    <!-- raising the command tracker logging level -->
+    <logger name="com.digitalasset.platform.apiserver.services.tracking.TrackerMap" level="WARN" />
+
+    <!--
+    explicitly enabling the info logging for sandbox main and ledger api server
+    for the usual startup info logging (which port, dar files, etc...)
+    -->
+    <logger name="com.digitalasset.platform.sandboxnext" level="INFO" />
```
Comment (SamirTalwar-DA) [inline ledger/participant-state/kvutils/app/src/main/resources/com/daml/ledger/participant/state/kvutils/app/logback.base.xml:31]: You almost certainly don't need this here.

File: ledger/participant-state/kvutils/app/src/main/resources/com/daml/ledger/participant/state/kvutils/app/logback.base.xml [hunk 3]
```diff
@@ -15,6 +15,19 @@
...
+    <logger name="com.digitalasset.platform.store.FlywayMigrations" level="WARN" />
+    <logger name="org.flywaydb" level="ERROR" />
+    <logger name="com.zaxxer.hikari" level="ERROR" />
+    <logger name="com.digitalasset.platform" level="WARN" />
+
+    <!-- raising the command tracker logging level -->
+    <logger name="com.digitalasset.platform.apiserver.services.tracking.TrackerMap" level="WARN" />
+
+    <!--
+    explicitly enabling the info logging for sandbox main and ledger api server
```
Comment (miklos-da) [inline ledger/participant-state/kvutils/app/src/main/resources/com/daml/ledger/participant/state/kvutils/app/logback.base.xml:28]: I don't see how this comment relates to the lines below. Can you clarify this, please?

Comment (gerolf-da) [reply ledger/participant-state/kvutils/app/src/main/resources/com/daml/ledger/participant/state/kvutils/app/logback.base.xml]: Thanks. Fixed.

**Comment by dasormeter** [comment]
@gerolf-da is this one gonna make it in the 1.0.1 patch, or will it be subsequent release after that?

**Comment by gerolf-da** [comment]
Will be in a subsequent release.

**Pull request merged**